modules/gapi/test/common/gapi_render_tests.hpp

```diff
@@ -0,0 +1,73 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_RENDER_TESTS_HPP
+#define OPENCV_GAPI_RENDER_TESTS_HPP
+
+#include "gapi_tests_common.hpp"
+#include "api/render_priv.hpp"
+
+namespace opencv_test
+{
+
+using Points            = std::vector<cv::Point>;
+using Rects             = std::vector<cv::Rect>;
+using PairOfPoints      = std::pair<cv::Point, cv::Point>;
+using VecOfPairOfPoints = std::vector<PairOfPoints>;
+
+template<class T>
+class RenderWithParam : public TestWithParam<T>
+{
+protected:
+    void Init()
+    {
+        MatType type = CV_8UC3;
+        out_mat_ocv  = cv::Mat(sz, type, cv::Scalar(0));
+        out_mat_gapi = cv::Mat(sz, type, cv::Scalar(0));
```

Comment:
Painting on a black image?
Probably painting on a white one (255) would be, hm, closer to the domain?

Reply:
Fixed